test/controllers/sessions_controller_test.rb

```diff
@@ -59,13 +59,13 @@ def test_password_checkparams_no_confirmpassword
   end
 
   def test_password_doesnotmatch 
```

Comment:
def test_with_no_mobileno_present_in_db 
    value = {mobile_no: **3456789012**, password: @user.password}
    response = post "/signin_users", value
    assert_equal(response.status,302)
  end


Faker

Reply:
def test_session_destroy
    response = delete "/signout_users"
    assert_equal(response.status,302)
  end




where will it redirect in this case?

Reply:
it will redirect to new_users_path